src/format.h

```diff
@@ -65,9 +65,29 @@ class Format {
            type_ == FormatType::kS8_UINT;
   }
 
+  uint32_t RowCount() const {
+    return static_cast<uint32_t>(components_.size());
+  }
+  uint32_t ColumnCount() const { return column_count_; }
+  void SetColumnCount(uint32_t c) { column_count_ = c; }
+
+  bool IsInt8() const { return AreAllComponents(FormatMode::kSInt, 8); }
```

Comment:
Since this unpacks vector types too, my expectation would be these would be called something like

IsElementInt16 etc.

I ... think I can live with what you have.  At least it keeps to the semantics of previous code.